services/order.py

```diff
@@ -0,0 +1,49 @@
+from django.contrib.auth import get_user_model
+from db.models import MovieSession, Ticket, Order
+from django.db import transaction
+from django.db.models import QuerySet
+
+
+def validate_movie_session(movie_session_id: int) -> bool:
+    try:
+        MovieSession.objects.get(id=movie_session_id)
+    except MovieSession.DoesNotExist:
+        print(f"MovieSession with id: {movie_session_id}, does not exist")
+
+    return True
+
+
+def create_order(
+        tickets: list[Ticket],
+        username: str,
+        date: str = None
+) -> None:
+
+    current_user = get_user_model().objects.get(username=username)
+    order = Order(user=current_user)

```

Comment:
there is no need to create additional variable here, u can make this directly in Order )

Reply:
In my opinion, this object provides less queries to the database. Also, when "data" is provided, we add it to our object below. And only then, in context manager, we save order to the database.